### PR TITLE
Updating message asking to wait between submissions 

### PR DESCRIPTION
**Pull request by EmperorRP**
# Descripción

In this issue, I change the message that asks to wait between code submissions

Fixes: #6467 

# Checklist:

- [x] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [] Se corrieron todas las pruebas y pasaron.
- [] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

File: frontend/templates/en.lang [hunk 1]
```diff
@@ -66,7 +66,7 @@ arenaRunSubmitFilesize = "The file size limit for uploaded files is %(limit)"
 arenaRunSubmitMissingLanguage = "You must choose a language"
 arenaRunSubmitPaste = "Paste your program's code here"
 arenaRunSubmitUpload = "Or upload the file"
-arenaRunSubmitWaitBetweenUploads = "You must wait %(submissionGap) seconds between submissions for a problem"
+arenaRunSubmitWaitBetweenUploads = "You have to wait %(submissionGap) seconds before you can make another submission to this problem."
```
Comment (pabo99) [inline frontend/templates/en.lang:69]: Cool.

Now, you need to modify the same string in the other available languages we have (`frontend/templates/es.lang`, `frontend/templates/pt.lang`). The new string in spanish was provided in the issue. In the case of portuguese language, you can use google translate to get the translation

Once you have modified those three files, you  need to run the following script:
```
./stuff/lint.sh
```
in order to generate extra needed files

Comment (EmperorRP) [reply frontend/templates/en.lang:69]: Alright, I made changes in Spanish and Portuguese, will run lint command in the git terminal soon

**Comment by EmperorRP** [comment]
Can you help me resolve this Python error

File: frontend/templates/es.lang [hunk 2]
```diff
@@ -66,7 +66,7 @@ arenaRunSubmitFilesize = "El límite para subir archivos son %(limit)"
 arenaRunSubmitMissingLanguage = "Debes elegir un lenguaje"
 arenaRunSubmitPaste = "Pega el código de tu programa aquí"
 arenaRunSubmitUpload = "O sube un archivo"
-arenaRunSubmitWaitBetweenUploads = "Debes esperar %(submissionGap) segundos entre cada envío del mismo problema"
+arenaRunSubmitWaitBetweenUploads = "Tienes que esperar %(submissionGap) segundos antes de puedas hacer otro envío a este problema."
```
Comment (lhchavez) [inline frontend/templates/es.lang:69]:
```suggestion
arenaRunSubmitWaitBetweenUploads = "Tienes que esperar %(submissionGap) segundos antes de que puedas hacer otro envío a este problema."
```

**Pull request closed without merging**